tests/test_sdk/conftest.py

```
@@ -0,0 +1,161 @@
+    
+    Expects MySQL running (via `make docker-test-services` or CI service).
+    """
+    url = os.getenv("TEST_MYSQL_URL", "mysql://root:root@localhost:3306/testdb")
+    
+    # Verify connection and create test schema
+    try:
+        import pymysql
+        conn = pymysql.connect(
+            host='localhost',
+            user='root',
+            password='root',
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)